examples/hitl/rearrange_v2/habitat_llm_loader.py

```diff
@@ -38,9 +43,21 @@ def load_collaboration_episode_data(
 
         return episode_data
 
+    def register_habitat_llm_extensions(config):
```

Comment:
A docstring would be nice. What do you mean by "extensions"? I guess this is habitat-llm extending habitat-lab actions/measures/sensors?